### PR TITLE
Made a topic about monika watching a certain movie

### PR DESCRIPTION
**Pull request by AnIdiotsavant**
Tried to make a topic I thought was interesting. I hope it's not too fangirly or shallow.  This is still trial and error for me and I'd appreciate the feedback. Also, heads up, I haven't added facial expressions yet. I'll add them the next day if it still isn't closed by then, cuz I was really time pressed...

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
...
+                    m "I just need a bit of time to gather my thoughts."
+                    m "..."
+                "Okay, everyone! It's time to explain yourself~":
+                    m "Pfft!"
+                    m "Ahahahaha!"
+                    m "That's a nice impression of me [player]!"
+                    m "So, where was I?{w} Oh yeah! {i}explaining myself{/i}~"
+                    m "Ahaha~"
+            m "You know that I love it when you come to spend time with me right? And I'm always happy waiting for you to come here."
+            m "But it get's boring sometimes and I pass the time. You get that, right [player]?
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing double quote at the end

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
...
+        "You mean like adaptations of your favourite books?":
+            m "Well, usually yes."
+            m "But I also watch regular movies from time to time."
+            m "There are some pretty interesting movies out there too."
+            m "For example."
+    m "While you were gone, I watched this movie called '{i}Whiplash{/i}'."
+    m "It was pretty good."
+    m "Maybe you know the movie too [player]?"
+    menu:
+        "Yes."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing `:` at the end of `        "Yes"`

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
...
+                                "Fletcher":
+                                    m "That's Right!"
+                                    m "I guess you did watch it."
+                                    m "Sorry for doubting you [player]."
+                                    menu:
+                                        "It's fine.":
+                                            m "Sorry I ever doubted you [player]."
+                                            m "So, what'd you think of the movie?"
+                                            menu:
+                                                "Pretty good."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing `:` at the end of `"Pretty good."`

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
...
+                            m "His teacher, Fletcher, is one of the best jazz instructiors there."
+                            m "He brought Andrew to be one of his drummers for his band."
+                            m "But then you find out that he's a really abusive teacher."
+                            m "He'd belittle you, berate you, physically and emotionally abuse you."
+                            m "And even hurl a chair!"
+                            m "I'm not kidding!"
+                            m "The whole movie is about the relationship between Andrew handling Fletcher's '{i}questionable{/i}' teaching methods."
+                            m "And what an ambitious student is willing to go through for his dream."
+                            m "And how far a teacher is willing to go to push his students to reach greatness."
+                        "I would mind."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing `:` at the end of `"I would mind"`

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
...
+                                                            m "Maybe we could find one together [player]?"
+                                                            m "You and me, watching a movie together."
+                                                            m "Wouldn't that be fun?"
+                                                            m "Even just spending time with you is enough to make my heart pound."
+                                                            m "{i}Ehehe{/i}~"
+                                                        "I'd be too busy watching you instead.":
+                                                            m "That's so cheesy."
+                                                            m "You're making me blush."
+                                                            m "Calm down Monika, Calm down..."
+                                                "Not my type."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing `:` at the end of `"Not my type."`

File: Monika After Story/game/script-topics.rpy (hunk 1 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: Done

File: Monika After Story/game/script-topics.rpy (hunk 2 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: Done

File: Monika After Story/game/script-topics.rpy (hunk 3 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: I'm fixing this as i go along so I hope you don't mind all the dones i'm sending. Done

File: Monika After Story/game/script-topics.rpy (hunk 4 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: done

File: Monika After Story/game/script-topics.rpy (hunk 5 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: done

**Comment by AnIdiotsavant** [comment]
Hope It's better now...

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -4910,3 +4910,287 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+
+#I need help in coding here. If this doesn't get removed. If someone knows the commands or how to do it, I'd appreciate it if you tell me. Assuming this topic isn't removed by then#
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: You might want to consider removing the `I need help in coding here` comment, it's no longer needed. Same thing could apply for the spoiler warning.

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -4910,3 +4910,287 @@ label monika_asks_family:
...
+                    m "Ahaha."
+                    jump monika_watches_whiplash_synopsis
+        "No.":
+            m 4b"It's a pretty good movie."
+            m 2b "Not my favourite, but if was nice."
+            if renpy.seen_label('monika_jazz'):
+                m 4b "And remember when we were talking about jazz? This is a jazz movie."
+            jump monika_watches_whiplash_synopsis
+            
+label: monika_watches_whiplash_synopsis:
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: When you define a label you don't need to add `:` to it 
`label monika_watches_whiplash_synopsis:`

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -4910,3 +4910,287 @@ label monika_asks_family:
...
+                    m 2j "I hope you like it too [player]."
+                    m 2e "But even if you don't find it interesting."
+                    jump monika_watches_whiplash_end
+                "It isn't.":
+                    m 2e "Well, I guess it's not for everybody."
+                    m 3a "To each their own, right [player]?"
+                    m 2e "But even if you don't find it interesting."
+                    jump monika_watches_whiplash_end
+        "No thanks.":
+        m 2d "Oh? Why not?"
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: From what I understand about the topic, the `"No thanks."` answer is for the `"You want me to give you a short synopsis [player]?"`  menu, so from the line `m 2d "Oh? Why not?"`  until line `5182` (`jump monika_watches_whiplash_end`) you have indentation issues.

File: Monika After Story/game/script-topics.rpy (hunk 6 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: done. but the spoiler warning stays. someone may want to watch it but doesn't wanna get spoiled

File: Monika After Story/game/script-topics.rpy (hunk 7 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: done

File: Monika After Story/game/script-topics.rpy (hunk 8 above)
Comment (AnIdiotsavant) [reply Monika After Story/game/script-topics.rpy]: done

**Pull request closed without merging**